car_rental/api/models.py

```diff
@@ -1,4 +1,8 @@
+from email.policy import default
+from pyexpat import model
+from random import choices
 from django.db import models
+from django.forms import FloatField
```

Comment:
FloatField is supported automatically in djangoDB right?

Reply:
yeah, it is...when I first used it, VS code automatically imported it for me for some reason.

Reply:
yeah could you try removing it and test if your code still works without this import? Then push your work